package.json

```diff
@@ -11,8 +11,9 @@
     "react-dom": "^17.0.2",
     "react-redux": "^7.2.4",
     "react-scripts": "4.0.3",
-    "recharts": "^2.0.9",
     "react-sliding-pane": "^7.0.0",
+    "react-spotify-auth": "^1.3.5",
```

Comment:
we don't use this package anymore because it doesn't give refresh tokens

Reply:
the package is a dependency inside one of the files, are we removing this now? It wouldn't build without this package

Reply:
good catch, 
I forgot to remove 
`import "react-spotify-auth/dist/index.css";`
in Login.jsx

You wanna remove it in this PR?